### DISCUSSION TITLE
Unable to download album on local setup

### DISCUSSION THREAD

**Discussion by Theiaz** [photoprism/photoprism#3924] [Q&A] [answered]
Hi,

I'm experimenting with Photoprism and it seems to be a nice piece of software!

I've setup my local instance with [docker compose](https://docs.photoprism.app/getting-started/docker-compose/) and tried to download some albums which I''ve created. However its just downloading an empty zip-folder.

Isn't this possible, is it?

**Comment by lastzero** [comment]
If you are downloading complete albums, they should be compressed while downloading without creation of a temporary zip file...

I recommend making sure you are using our latest stable release and that the problem is not caused by any CPU or memory limitations, for example because the server is stopped/killed by the operating system. Reverse proxies like Nginx might also cause such issues.

**Reply by lastzero** [reply]
Also see:
- https://docs.photoprism.app/getting-started/troubleshooting/#zip-archives
- https://docs.photoprism.app/user-guide/settings/general/#downloads

**Reply by Theiaz** [reply]
Thanks for the fast response!

I do not use a reverse proxy at all, I'm just running photoprism locally on localhost:2342 starting with `docker-compose up -d`. I'm using the latest docker-image.
 
Process is still running, however photoprism is indexing at the moment, which results in high cpu/memory usage. Maybe this causes the issue?

**Reply by lastzero** [reply, marked as answer]
Possibly. It could also still be a filesystem permission problem or the server is running out of space on one partition. Hard to tell with the information available.

**Reply by lastzero** [reply]
You could also try a different browser to see if that helps...

**Reply by Theiaz** [reply]
Seems to work now, thanks for the help!
